### DISCUSSION TITLE
v2.2.1.6 Verus Mining

### DISCUSSION THREAD

**Discussion by skippy2481** [romslf/Ultimate-Proxy#19] [Q&A]
I just started running ultimate proxy on a vps and I have 10 mobile phones and 4 orangepi 5 atm mining but it seems that there not turning in shares like I would think I'm not sure but I thing that it may have something to do with having the phones mixed with the orange pi's it changes the difficultity a lot and on the higher difficulity the orange pis seem to get more shares than the phones any thoguhts?

**Comment by skippy2481** [comment]
I have another question I am hasving trouble trying to get shares accepted on us.vipor.net for verus coin I have connected TCP and SSL the miners find plenty of shares but none acceptedthe diffuility sets at 35.662mh/s and on other pools it goes about 1.5th/s

**Comment by romslf** [comment]
> I just started running ultimate proxy on a vps and I have 10 mobile phones and 4 orangepi 5 atm mining but it seems that there not turning in shares like I would think I'm not sure but I thing that it may have something to do with having the phones mixed with the orange pi's it changes the difficultity a lot and on the higher difficulity the orange pis seem to get more shares than the phones any thoguhts?

Hey 👋 
Welcome to Ultimate Proxy !

That's the excepted behaviour since the proxy connect only once to the pool/act as one big worker, so your smaller miners will find less shares than your bigger ones since they are all assigned the same difficulty.

As for vipor pool, I can't say for sure what's the issue here, I will need more informations or give it a shot by myself

**Reply by skippy2481** [reply]
I have plenty of vps's to be able to run a proxy on do you think it would be a better option to run the orange pi's on one proxy and the phones on another and I am using tmux to keep it running when I close my session window out would they be a better option of what do you think

**Reply by romslf** [reply]
> I have plenty of vps's to be able to run a proxy on do you think it would be a better option to run the orange pi's on one proxy and the phones on another and I am using tmux to keep it running when I close my session window out would they be a better option of what do you think

Yeah you can do that if you want, but it shouldn't impact your earnings

I recommend using [PM2](https://pm2.keymetrics.io/docs/usage/quick-start/), so it auto restart on crash or computer reboot

**Reply by skippy2481** [reply]
I am mining verus on luckpool and I have a lot of rejected shares are there anyway to get them lower it seems to be alot to me compared to mining without a proxy don't worry about the s21 at the top I only have one and its off right now but it stays on the screen for some reason
![Screenshot from 2024-05-21 04-39-22](https://github.com/romslf/Ultimate-Proxy/assets/108590997/0eec74e2-c1cc-474a-b142-326fac08b47e)

**Reply by romslf** [reply]
Try using a static difficulty by using "d=16384S" as password

**Reply by skippy2481** [reply]
Ok I set the difficulty as you said on the proxy's config.json and the miners I left the way they were I never set them and here is some screenshots on the pool and the proxy I and running this on a vps provider instead of at home I still have more rejected shares than I would like and why does it have to reconnect all the time is that normal for the proxy or does that have something to do with the dev fee what else can I do to help this and 9:00am on the pool time is when I switched the difficulty  and you can see how the large hashrate swings smoothed out, thanks--- and I restarted the proxy so the stats are new   and another thing is there a way to get the reported hashrate to work correctly
![Screenshot from 2024-05-25 11-31-13](https://github.com/romslf/Ultimate-Proxy/assets/108590997/8d970b0e-8cbd-4e66-8edd-c37308ca50f7)
![Screenshot from 2024-05-25 11-33-12](https://github.com/romslf/Ultimate-Proxy/assets/108590997/5b62da51-0f17-4201-9de9-99b4fcd2fd71)
